docs/index.rst

```diff
@@ -45,7 +45,7 @@ Write a test case ``backup_test.py`` describing the expected behavior:
         .and_assert_called_once()
       backup.Backup().delete('/file/to/delete')
 
-TestSlide's :doc:`strict_mock/index` , :doc:`mock_callable/index` and :doc:`mock_constructor/index` are seamlessly integrated with Python's TestCase.
+TestSlide's :doc:`strict_mock/index`, :doc:`matchers/index`  , :doc:`mock_callable/index` and :doc:`mock_constructor/index` are seamlessly integrated with Python's TestCase.
```

Comment:
I don't think matchers are used on the previous example, we can probably skip this.